### ISSUE TITLE
Consider adding Jami (formerly known as GNU Ring)

### ISSUE THREAD

**Issue by OpenSourceAnarchist** [clearlinux/distribution#501]
**Is your feature request related to a problem? Please describe.**
Not related to a problem _per se_, but having an alternative to Skype that is part of the GNU Project would be great for desktop use where video/audio calling is necessary.

**Describe the solution you'd like**
An attempt to package Jami. Skype for Web is the current software I can use, and having a native client that is free and open-source would be great.

**Describe alternatives you've considered**
Documentation on how to install the proprietary Skype application, but this seems more difficult and not in-line with Clear's goals. Continuing to use Skype for Web would work too, but that is not supporting the goals of the GNU Project.

**Additional context**
The website of Jami can be found here: [https://jami.net/](https://jami.net/)

**Comment by ahkok** [comment]
A quick look at the build instructions doesn't bode well.

**Comment by fenrus75** [comment]
they seem to do their utmost best to hide where their source code is
downloadable -(

**Comment by OpenSourceAnarchist** [comment]
Yeah I thought it might be difficult but was hoping that was not the case. I know it is probably not of much help, but Arch Linux has several PKGBUILDS available that may be of use.

opendht: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/opendht

ring-daemon: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/ring-daemon

libringclient: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/libringclient

ring-gnome: https://git.archlinux.org/svntogit/community.git/tree/trunk?h=packages/ring-gnome

ring-kde: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/ring-kde

If it is too difficult to get working, I understand completely. I can continue to use Ubuntu, Fedora, or Arch alongside Clear for these obscure cases. Really appreciate the work and dedication of the Clear dev. team!
